jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Attributes.java

```diff
@@ -609,5 +761,17 @@ public Map<String, Object> asAttributeMap()
         {
             return Collections.emptyMap();
         }
+
+        @Override
+        public int hashCode()
+        {
+            return Attributes.hashCode(this);
+        }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            return o instanceof Attributes && Attributes.equals(this, o);
+        }
```

Comment:
These are not necessary, given how `Wrapper` has been implemented (by not delegating).

Reply:
tell intellij that